step-templates/iis-website-set-ip-security.json

```diff
@@ -60,7 +60,7 @@
       "Links": {}
     }
   ],
-  "LastModifiedBy": "perosb",
+  "LastModifiedBy": "Graham King",
```

Comment:
If you could update this to your GitHub handle, it will then be correctly linked on the Library site